Gosu/Audio.hpp

```diff
@@ -136,6 +136,20 @@ namespace Gosu
         double volume() const;
         //! \param volume Can be anything from 0.0 (silence) to 1.0 (full volume).
         void set_volume(double volume);
+        //! Returns the current song's playback position in seconds
+        int position() const;
+        //! Returns the current song's playback position in minutes
+        int position_minutes();
+        //! Returns the current song's playback position in hours
+        int position_hours();
+        //! \param pos position can be any value from 0.0 through the last second
+        void set_position(float pos);
+        //! Returns how long your song might be
+        float duration() const;
```

Comment:
Having too many representations for times (int seconds, unsigned long Gosu::milliseconds, float duration) is dangerous because we don't have any type safety here or in Ruby.
In my opinion, having a floating-point number of seconds is the best approach. However it should be double, not float, for consistency with `double volume()` and all of the rendering APIs, which use doubles.